repo/packages/A/artifactory-primary/2/config.json

```diff
@@ -0,0 +1,95 @@
+{
+  "type": "object",
+  "properties": {
+    "artifactory-primary": {
+      "description": "Artifactory framework configuration properties",
+      "type": "object",
+      "properties": {
+        "framework-name": {
+          "description": "The framework name to register with Mesos.",
+          "type": "string",
+          "default": "artifactory-primary"
+        },
+        "cpus": {
+          "description": "CPU shares to allocate to each Artifactory master.",
+          "type": "number",
+          "default": 2.0,
+          "minimum": 1.0
+        },
+        "mem": {
+          "description": "Memory to allocate to each Artifactory master.",
+          "type": "number",
+          "default": 2048.0,
+          "minimum": 1024.0
+        },
+        "high-availability": {
+          "description": "High availability mode properties.",
+          "type": "object",
+          "properties": {
+            "enabled": {
+              "description": "Run Artifactory in high availability mode.",
+              "type": "boolean",
+              "default": false
+            },
+            "nodes": {
+              "description": "The number of nodes to run in high availability mode.",
```

Comment:
Is this setting useful for the primary node? My understanding was that there is just a single primary.